custom/icds_reports/utils/fix_child_health_missing_month.sql

```diff
@@ -0,0 +1,613 @@
+INSERT INTO "icds_dashboard_growth_monitoring_forms" (
```

Comment:
@calellowitz  Sory, wanted to ask you opion on this query. But mistakenly made that comment on other Pr.  But asking here as well, can this query be optimized further?

Reply:
I think you can use the previous month's child health monthly as a replacement for the agg growth monitoring table in the join (should have the same data since it is inserted from there into child health monthly) and then you will only have to look through one month of gm forms. It won't be great still, but it is likely far better.

Reply:
I dont know if this query will work as it is written. You will need to create the table to insert data into since that child table has long since been dropped. Since this is a one off command, that table can be standalone, there is no need to make it a child of our existing intermediate table (and if you make it `unlogged` the insert will be a lot faster). That temp table should be distributed as well.